src/lib/crypto/krb/t_fortuna.c

```diff
@@ -102,7 +102,7 @@ head_tail_test(struct fortuna_state *st)
     }
 
     for (bit = 0; bit < 8; bit++) {
-        res = ((double)abs(len - bits[bit] * 2)) / (double)len;
+        res = ((double)(len - bits[bit] * 2)) / (double)len;
```

Comment:
Okay, I ran this code through a debugger.  For bit 1 (the second iteration of the loop), bits[bit] * 2 is indeed slightly larger than len, so the expression (len - bits[bit] * 2) evaluates to a large size_t value.  But since abs takes an int argument, the value is coerced to -1090 and the result of abs() is 1090.  What is the clang warning here?

The change in the PR causes t_fortuna to print an error message and exit with status 1.  However, because of the way Makefile.in runs t_fortuna, it doesn't notice (it only notices that stdout matched the expected output) so the Travis tests pass.

Simply making i and len have type ``int`` appears to be a better fix.